hlx_statics/scripts/scripts.js

```diff
@@ -336,13 +312,60 @@ export async function loadIms() {
   return window.imsLoaded;
 }
 
+/**
+ * Load config items into the window for use
+ */
+function loadConfig() {
+  window.REDOCLY = `eyJ0IjpmYWxzZSwiaSI6MTczMjEzNzQzNSwiZSI6MTc1OTI2NTQxNywiaCI6WyJyZWRvYy5seSIsImRldmVsb3Blci5hZG9iZS5jb20iLCJkZXZlbG9wZXItc3RhZ2UuYWRvYmUuY29tIiwiZGV2ZWxvcGVyLmZyYW1lLmlvIiwiZGV2ZWxvcGVyLmRldi5mcmFtZS5pbyIsImxvY2FsaG9zdC5jb3JwLmFkb2JlLmNvbSIsInJlZG9jbHktYXBpLWJsb2NrLS1hZHAtZGV2c2l0ZS0tYWRvYmVkb2NzLmFlbS5wYWdlIiwiZGV2ZWxvcGVyLWRldi5hZG9iZS5jb20iXSwicyI6InBvcnRhbCJ9.gf0tCrK+ApckZEqbuOlYJFlt19NU6UEWpiruC4VIMg9ZYUojkyDGde2aEKpBK2cm57r6yNNFNWHyIRljWAQnsg==`;
+
+  // check to see if we're on an aem url, stage or prod
+  if (isHlxPath(window.location.host)) {
+    window.marketingtech = {
+      adobe: {
+        launch: {
+          property: 'global',
+          environment: 'dev',
+        },
+        analytics: {
+          additionalAccounts: 'pgeo1xxpnwadobeio-qa',
+        },
+      },
+    };
+  } else if (!isHlxPath(window.location.host) && isStageEnvironment(window.location.host)) { 
+    window.marketingtech = {
+      adobe: {
+        launch: {
+          property: 'global',
+          environment: 'dev',
```

Comment:
Should this be stage? e.g.

```suggestion
          environment: 'stage',
```

Reply:
Actually it needs to be set to `dev` in this instance...